test/sql/upsert/insert_or_replace_ambiguity.test

```diff
@@ -0,0 +1,70 @@
+# name: test/sql/upsert/insert_or_replace_ambiguity.test
+# group: [upsert]
+
+# Single primary key on multiple columns
+statement ok
+create table foo(
+	a int,
+	b int,
+	c int,
+	primary key(a,b,c)
+);
+
+statement ok
+insert or replace into foo values (1,2,3);
```

Comment:
Can we perform the inserts twice and also check the table data after? And let's do the same for the other tests.